engine/security-api/src/main/scala/pl/touk/nussknacker/ui/security/api/AuthenticationConfiguration.scala

```diff
@@ -0,0 +1,62 @@
+package pl.touk.nussknacker.ui.security.api
+
+import java.io.File
+import java.net.URI
+
+import com.typesafe.config.{Config, ConfigFactory}
+import com.typesafe.scalalogging.LazyLogging
+import net.ceedubs.ficus.readers.ValueReader
+import pl.touk.nussknacker.ui.security.api.AuthenticationMethod.AuthenticationMethod
+import pl.touk.nussknacker.ui.security.api.DefaultAuthenticationConfiguration.DefaultConfigUser
+import pl.touk.nussknacker.ui.security.api.GlobalPermission.GlobalPermission
+import pl.touk.nussknacker.ui.security.api.Permission.Permission
+
+trait AuthenticationConfiguration extends Product {
```

Comment:
extends Product is not really needed here?

Reply:
Removed.